android/src/main/kotlin/com/signify/hue/flutterreactiveble/channelhandlers/ScanDevicesHandler.kt

```diff
@@ -67,4 +70,4 @@ class ScanDevicesHandler(private val bleClient: com.signify.hue.flutterreactiveb
     }
 }
 
-private data class ScanParameters(val filter: List<ParcelUuid>, val mode: ScanMode, val locationServiceIsMandatory: Boolean)
+internal data class ScanParameters(val filter: List<ParcelUuid>, val mode: ScanMode, val locationServiceIsMandatory: Boolean)
```

Comment:
Are you sure `private` doesn't work here? I suspect you needed `internal` with the previous approach.

Reply:
Good catch fixed it!